src/lib/data/createables.ts

```diff
@@ -2113,6 +2113,48 @@ const Createables: Createable[] = [
 		}),
 		outputItems: new Bank().add('Celestial signet')
 	},
+	{
+		name: "Xeric's talisman",
+		inputItems: new Bank({
+			"Xeric's talisman (inert)": 1,
+			'Lizardman fang': 1000
+		}),
+		outputItems: new Bank({
+			"Xeric's talisman": 1
+		})
+	},
```

Comment:
this could be a chargeable item too

Reply:
Did have a go at making it mountable in PoH but couldnt get it working, will look at it again at a later date but no need to hinder the rest of this pr on it.